tests/modules/vardictjava/nextflow.config

```diff
@@ -0,0 +1,9 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+    
+    withName: VARDICTJAVA {
```

Comment:
this is only necessary when you want to add extra args during the test, but can stay